lib/matplotlib/pyplot.py

```diff
@@ -2928,8 +2934,9 @@ def cohere(
     NFFT: int = 256,
     Fs: float = 2,
     Fc: int = 0,
-    detrend: Literal["none", "mean", "linear"]
-    | Callable[[ArrayLike], ArrayLike] = mlab.detrend_none,
+    detrend: (
```

Comment:
This and the following changes are formatting only and unrelated to the topic.

Reply:
This happened after I ran boilerplate.py to get the label to update. I wasn't sure what to do with all the formatting changes so I left them.

Reply:
I'm unclear why this happened. I've run boilerplate.py against main and there were no changes. If there was, I would have made a separate PR so that it's getting fixed quickly for all.

Reply:
This also happened in my other PR but I changed it back manually and managed to keep it off the commits.

Reply:
boilerplate.py is using black, so possibly the difference is caused by having a different black version or if you have any local black config settings?

Reply:
That must be it!